contrib/IECoreAlembic/src/IECoreAlembic/AlembicScene.cpp

```diff
@@ -720,7 +976,7 @@ void AlembicScene::readTags( NameList &tags, int filter ) const
 
 void AlembicScene::writeTags( const NameList &tags )
 {
-	throw IECore::NotImplementedException( "AlembicScene::writeTags" );
+	IECore::msg( IECore::Msg::Warning, "AlembicScene::writeAttribute", "Not implemented" );
```

Comment:
Yep. Sets map quite naturally to AbcCollections, so that would be trivial to implement. Mapping tags to collections and back is more of a pain though (we'd end up wanting PathMatcher in Cortex to do it probably), so I'd like to avoid having to implement it only to throw it away when we move to sets in SceneInterface. I'm also note sure what other apps support AbcCollections - as far as I can tell there's no way of exporting one out of Maya, for instance.